### DISCUSSION TITLE
are magic comments or ways to manipulate how the components are transpiled to another framework?

### DISCUSSION THREAD

**Discussion by rssilvaba** [BuilderIO/jsx-lite#1006] [Q&A] [answered]
Hi, angular has the annoying feature of wrapping everything, even when you use a fragment in jsx, it will translate to ng-container which does not avoid the component name wrapper.

are there any ways to avoid that in mitosis or for example instruct mistoses to use a attr selector for the component instead of a element custom name?

**Comment by samijaber** [comment, marked as answer]
It depends on the specific way you want to manipulate the output. We have a customizable plugin system that lets you alter the JSON schema of the component, and the string output of generators: https://github.com/BuilderIO/mitosis/blob/main/docs/customizability.md

If the limitation you're encountering cannot be fixed by a simple user-land plugin, we are always open to having a contribution or change made to the Angular generator itself, in this case as a configuration option (`hasWrapper: boolean`) passed to the Angular generator.

**Reply by rssilvaba** [reply]
Thanks, hard to say if it would be something that requires a flag as you pointed. I just got to know mitosis.
But you gave a good starting point. I will look into it and come back to post my findings.
